pkg/pr/utils.go

```diff
@@ -40,3 +40,18 @@ func replaceTo(from, to string, rep func(data []byte) ([]byte, error)) error {
 func replaceInPlace(path string, rep func(data []byte) ([]byte, error)) error {
 	return replaceTo(path, path, rep)
 }
+
+func removeMatches(glob string) error {
+	matches, err := filepath.Glob(glob)
```

Comment:
yep, if full filepath is provided, `filepath.Glob` checks for file existence and returns it as part of `matches` array if it exists, otherwise it returns `null` and nothing happens.

Reply:
cool that works then